chapter-05/1_1_types/money/amount.go

```diff
@@ -0,0 +1,7 @@
+package money
+
+// Amount represents a given amount of money.
+type Amount struct {
+	number   Number
```

Comment:
Amount needs to be exposed (returned by Convert), but why would Number and Currenecy be?